pinc/archiving.inc

```diff
@@ -223,30 +264,3 @@ $archival_recovery_multiplier = 1;
 // If you're running the script when the site is down (e.g., when upgrading),
 // you probably don't need to insert sleeps, so you can eliminate them
 // by setting this variable to zero.
-
-// -----------------------------------------------------------------------------
-
-function archival_skip_logs_init()
```

Comment:
Thanks for catching and deleting these!

Reply:
It's easier to delete than to add DocBlock comments :)